core/templates/dev/head/exploration_editor/EditorServices.js

```diff
@@ -673,6 +673,65 @@ oppia.factory('explorationStatesService', [
       editorContextService, validatorsService, newStateTemplateService,
       explorationGadgetsService) {
     var _states = null;
+
+    // Maps backend names to the corresponding frontend dict accessor lists.
+    var PROPERTY_REF_DATA = {
```

Comment:
I wish angular had scoped constants. Alas, let's keep it local. I agree it shouldn't be accessible in general like constants are.